core/templates/pages/topic-editor-page/modal-templates/questions-opportunities-select-difficulty-modal.component.html

```diff
@@ -1,36 +1,24 @@
-<div class="oppia-outer"
-     *ngFor="let skillWithDifficulty of linkedSkillsWithDifficulty; trackBy: index">
-  <div class="modal-header">
-    <h3 *ngIf="linkedSkillsWithDifficulty.length === 0">
-      Select skills you would like to link to the question.
-    </h3>
-    <h3 class="oppia-modal-header" *ngIf="linkedSkillsWithDifficulty.length > 0">
-      {{ skillWithDifficulty.getDescription() }}
-    </h3>
-    <button type="button"
-            class="close oppia-close-button-position"
-            (click)="cancel()"
-            aria-label="Close">
-      <span aria-hidden="true">&times;</span>
-    </button>
+<div class="modal-header">
```

Comment:
Suggest showing a video to the "proof" section to verify this is working properly.

Reply:
No big changes to the file.